PatientWebApplication/FeedbackMicroserviceApi/Service/FeedbackService.cs

```diff
@@ -0,0 +1,92 @@
+﻿using FeedbackMicroserviceApi.Adapters;
+using FeedbackMicroserviceApi.DbContextModel;
+using FeedbackMicroserviceApi.Dtos;
+using FeedbackMicroserviceApi.Model;
+using FeedbackMicroserviceApi.Repository;
+using System.Collections.Generic;
+
+namespace FeedbackMicroserviceApi.Service
+{
+    /// <summary>Class <c>FeedbackService</c> handles feedback business logic.
+    /// </summary>
+    public class FeedbackService
+    {
+        /// <value>Property <c>FeedbackRepository</c> represents the repository used for data access.</value>
+        private FeedbackRepository FeedbackRepository { get; set; }
+        /// <summary>This constructor injects the FeedbackService with matching FeedbackRepository.</summary>
+        /// <param name="context"><c>context</c> is type of <c>DbContext</c>, and it's used for accessing MYSQL database.</param>
+        public FeedbackService(MyDbContext context)
+        {
+            FeedbackRepository = new FeedbackRepository(context);
+        }
+
+
+        /// <summary> This method converts <paramref name="dto"/> to <c>Feedback</c> using <c>FeedbackAdapter</c> and sends it to <c>FeedbackRepository</c>. </summary>
+        /// <param name="dto"><c>dto</c> is Data Transfer Object of a <c>Feedback</c> that contains <c>Message</c>, <c>IsPublic</c>, <c>IsAnonymous</c> and <c>PatientId</c>. 
+        /// </param>
+        /// <returns>if patient exists returns successfully created feedback; otherwise, return <c>null</c></returns>
+        public Feedback Create(FeedbackDto dto)
+        {
+            Feedback feedback = FeedbackAdapter.FeedbackDtoToFeedback(dto);
+            // Feedback feedback = HttpRequests.CreateFeedback(dto).Result;
+            return FeedbackRepository.Add(feedback);
```

Comment:
Write this in form:
"
return FeedbackRepository.Add(FeedbackAdapter.FeedbackDtoToFeedback(dto));
"
and delete comment